src/status_im/data_store/user_statuses.cljs

```diff
@@ -40,7 +40,8 @@
 
 (defn save-statuses-tx
   "Returns tx function for saving message user statuses"
-  [user-statuses]
-  (fn [realm]
-    (doseq [user-status user-statuses]
-      ((save-status-tx user-status) realm))))
+  ([user-statuses]
```

Comment:
Parentheses are not necessary here (only needed if multiple signatures are provided)

Reply:
fixed